maven-wrapper/pom.xml

```diff
@@ -40,6 +40,12 @@ under the License.
       <version>4.13.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.stefanbirkner</groupId>
+      <artifactId>system-rules</artifactId>
+      <version>1.19.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I see that is licensed on CPL 1.0
It is CATEGORY B - https://www.apache.org/legal/resolved.html#category-b
I don't want to consider a specified conditions for it.

for test purpose we can add method eg. getEnv(key) to Installer and next use spy or override method in test

Reply:
It’s a test dependency, so it won’t end up in the distribution. Or does a test dependency still count as ‘included in the product’?

Reply:
> I see that is licensed on CPL 1.0 It is CATEGORY B - https://www.apache.org/legal/resolved.html#category-b I don't want to consider a specified conditions for it.
> 
> for test purpose we can add method eg. getEnv(key) to Installer and next use spy or override method in test

Here we go.